src/mui/list/Actions.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { CardActions } from 'material-ui/Card';
+import FlatButton from 'material-ui/FlatButton';
+import NavigationRefresh from 'material-ui/svg-icons/navigation/refresh';
+import CreateButton from '../button/CreateButton';
+
+const cardActionStyle = {
+    zIndex: 2,
+    display: 'inline-block',
+    float: 'right',
+};
+
+const Actions = ({ resource, Filter, displayedFilters, filterValues, hasCreate, basePath, showFilter, refresh }) => (
```

Comment:
I agree `<Filter ... />` is better than `<filter ... />` inside a component rendering code but it looks weird as a prop. Conventions are to name the prop in camelCase (`filter`) and you can use es6 to rename it inside the component with:

```js
const Actions = ({ resource, filter: Filter, displayedFilters, filterValues, hasCreate, basePath, showFilter, refresh }) => (
```